api/src/test/java/io/tracee/TraceeTest.java

```diff
@@ -35,6 +35,7 @@ public void backendRetrievalShouldThrowTraceeExceptionWithDefaultResolver() {
 		}
 	}
 
+	@SuppressWarnings("unchecked")
```

Comment:
Please change Line 43 to `when(testBackendProvider.getBackendProviders()).thenThrow(new RuntimeException());` instead.